pallets/bridge/src/lib.rs

```diff
@@ -285,12 +253,6 @@ pub mod pallet {
 	pub enum Error<T> {
 		/// Invalid transfer
 		InvalidTransfer,
-
-		/// Not enough means for performing a transfer
-		InsufficientBalance,
```

Comment:
Removed because now is used the own `InsufficienteBalance` from `pallet-balance`.